README.md

```diff
@@ -1,25 +1,26 @@
-# Watchmarket Blockatlas version
-Is under development at master branch. Was migrated from [Blockatlas](https://github.com/trustwallet/blockatlas) at version/blockatlas branch
+# Watchmarket
 
 [![Build Status](https://dev.azure.com/TrustWallet/WatchMarket/_apis/build/status/trustwallet.watchmarket?branchName=master)](https://dev.azure.com/TrustWallet/WatchMarket/_build/latest?definitionId=45&branchName=master)
 [![codecov](https://codecov.io/gh/trustwallet/watchmarket/branch/develop/graph/badge.svg)](https://codecov.io/gh/trustwallet/watchmarket)
 
-**IMPORTANT!**
-Is under development at master branch. Was migrated from [Blockatlas](https://github.com/trustwallet/blockatlas) at version/blockatlas branch
+> Watchmarket is a Blockchain explorer API aggregator and caching layer. It's your one-stop-shop to get information for (almost) any coin in a common format
 
-### What is “Watchmarket”?
-Watchmarket is an aggregation and caching service for blockchain market information. 
-The main features of it are:
-1. **Aggregation**: it standardizes information from different blockchain data providers e.g. Coinmarketcap into a unified format
-2. **Caching**: it acts as a caching layer for this information mainly catering to the goal of cost savings as API calls to data providers are expensive
 
-### Architecture
+Watchmarket comes with three apps:
+* API: RESTful API to retrieve coin info, charts, and tickers
+* Observer: caches data from explorer APIs in Redis
+* Swapper: API explorer
 
-This project consists of 2 main parts: Observer and REST API 
+#### Supported Explorer APIs
 
-Observer - service that periodically fetch latest data from **data providers** (like coinmarketcap), parse it to the common data structure, set the parsed data to the cache (Redis)
+<a href="https://coinmarketcap.com" target="_blank"><img src="https://coinmarketcap.com/apple-touch-icon.png" width="32" /></a>
+<a href="https://www.binance.org/" target="_blank"><img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/binance/info/logo.png" width="32" /></a>
+<a href="https://compound.finance/" target="_blank"><img src="https://assets.coingecko.com/markets/images/517/large/compound-finance.png" width="32" /></a>
+<a href="https://fixer.io/" target="_blank"><img src="https://fixer.io/fixer_images/fixer_money.png" width="32" /></a>
+<a href="https://www.coingecko.com/" target="_blank"><img src="https://static.coingecko.com/s/thumbnail-007177f3eca19695592f0b8b0eabbdae282b54154e1be912285c9034ea6cbaf2.png" width="32" /></a>
 
-REST API - allows to get cached data through REST HTTP API
+**FYI!**
+Watchmarket was recently spun out of [Blockatlas](https://github.com/trustwallet/blockatlas) which remains under branch `version/blockatlas`.
```

Comment:
Maybe replace with actual date so in future "recently" will change it's meaning.